productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Productive seas fish and shellfish stocks fished sustainably",
+    "publisher": "https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs",
+    "creator": "https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs",
+    "summary": "The proportion of stocks fished sustainably in both the long term and in recent years",
```

Comment:
Replace with:
"Proportion of commercial fish and shellfish stocks that are within safe biological limits and fished sustainably"

Reply:
adjusted as requested